src/globals/scss/core/_typography.scss

```diff
@@ -154,41 +123,37 @@
   // Add a negative top margin to headings that appear directly after other
   // headings
 
-  .govuk-heading-l + .govuk-heading-m {
-    margin-top: -$govuk-spacing-scale-1;
-  }
-
-  .govuk-heading-m + .govuk-heading-s {
-    margin-top: -$govuk-spacing-scale-2;
-  }
+  // .govuk-heading-l + .govuk-heading-m {
```

Comment:
Let's not leave commented out code in here – there's a good chance it'll just hang around forever. Delete it - if we need to bring it back, that's what version control is for 🙂

Reply:
👍  Code taken out.